integration/suite.rb

```diff
@@ -72,11 +72,7 @@ def pink
       puts "#{test_name} #{'SUCCEEDED'.green}"
     ensure
       if options[:vm_maintenance] && succeeded
-        puts "Halting VM".yellow
-        unless system("vagrant halt")
-          puts "Tried to halt #{test_name} but it failed".red
-        end
-        puts "Halted VM"
+        
```

Comment:
I admit there are different levels of "cleaning up," but leaving a VM running in the background seems like a bad thing to do. It sounds like you were using the integration test's VM as a quick way to launch a Consul instance. That's great for development, but it shouldn't be the preferred way to do integration tests.  Maybe we should set up a separate Vagrant file for live testing while writing new features?